src/index.css

```diff
@@ -1,13 +1,27 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
     sans-serif;
```

Comment:
- [ ] According to the design guidelines, you're expected to use the Lato font as your body font for this project. Kindly use the [Google Lato](https://fonts.google.com/specimen/Lato?query=Lato) font family for your project as required 👍🏽

<img width="1397" alt="Screenshot 2023-03-24 at 9 01 23 AM" src="https://user-images.githubusercontent.com/50941074/227474271-cde56a97-464c-4322-877c-3129813f494d.png">